docs/pages/guides/icons.md

```diff
@@ -2,227 +2,85 @@
 title: Icons
 ---
 
-import SnackInline from '~/components/plugins/SnackInline';
+import SnackEmbed from '~/components/plugins/SnackEmbed';
 
-As trendy as it is these days, not every app has to use emoji for all icons -- maybe you want to pull in a popular set through an icon font like FontAwesome, Glyphicons or Ionicons, or you just use some PNGs that you carefully picked out on [The Noun Project](https://thenounproject.com/). Let's look at how to do both of these approaches.
+As trendy as it is these days, not every app has to use emojis for all icons -- maybe you want to pull in a popular set through an icon font like FontAwesome, Glyphicons or Ionicons, or you just use some PNGs that you carefully picked out on [The Noun Project](https://thenounproject.com/). Let's look at how to do both of these approaches.
 
 ## @expo/vector-icons
 
-This library is installed by default on the template project that get through `npx create-expo-app` -- it is part of the `expo` package. It includes popular icon sets and you can browse all of the icons using [icons.expo.fyi](https://icons.expo.fyi).
+This library is installed by default on the template project using `npx create-expo-app` and is part of the `expo` package. It is built on top of [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons) and uses a similar API. It includes popular icon sets that you can browse at [icons.expo.fyi](https://icons.expo.fyi).
 
-<SnackInline label='Vector icons' dependencies={['@expo/vector-icons']}>
+In the example below, the component loads the `Ionicons` font, and renders a checkmark icon.
 
-<!-- prettier-ignore -->
-```jsx
-import * as React from 'react';
-import { View, StyleSheet } from 'react-native';
-import Ionicons from '@expo/vector-icons/Ionicons';
-
-export default function App() {
-  return (
-    <View style={styles.container}>
-      /* @info */
-      <Ionicons name="md-checkmark-circle" size={32} color="green" />/* @end */
+<SnackEmbed snackId="@amanhimself/expo-vector-icons-ionicons-example" preview platform="web" />
```

Comment:
It is preferred to use incline code instead of `snackID` (especially private one), since only you will be able to edit it's code without changing the URL.

Reply:
@Simek, are you suggesting using something like this:

```markdown
<SnackEmbed platform="web" preview name="Default layout">
{`...snack code snippet`}
</SnackEmbed>
``` 

I was just trying to follow Jon's and Brent's example 😅 but at the same time wondering, using them from personal account doesn't feel right 🙈

Reply:
Yup, it would be nice to switch to the `SnackInline` component (one used before changes) and place new code inside it as you have mentioned above. Using inline Snacks also allow us to highlight the part of the code if needed.

Reply:
@Simek, reverted back and changed to using SnackInline component